src/Sarif.Converters/TSLintLoader.cs

```diff
@@ -73,8 +73,26 @@ public TSLintLog ReadLog(Stream input)
         // "fix":[{"innerStart":4429,"innerLength":0,"innerText":"\r\n"}]
         // "fix":[{"innerStart":4429,"innerLength":0,"innerText":"\r\n"},{"innerStart":4429,"innerLength":0,"innerText":"\r\n"}]
         //
+        // The following pattern also occurs, although the most recent version of the TSLint
+        // source code does not appear to support it:
+        //
+        // "fix": {
+        //   "innerRuleName": "no-trailing-whitespace",
```

Comment:
I might be wrong, but from the code below it looks like we're throwing innerRuleName away. Is that intended?

Reply:
@boAndron Yes, we don't have a slot for that.

Reply:
Got it, thanks!